src/hotspot/share/utilities/ostream.hpp

```diff
@@ -86,7 +86,6 @@ class outputStream : public ResourceObj {
    void move_to(int col, int slop = 6, int min_space = 2);
 
    // sizing
-   int width()    const { return _width;    }
    int position() const { return _position; }
    julong count() const { return _precount + _position; }
    void set_count(julong count) { _precount = count - _position; }
```

Comment:
These julong should probably be uint64_t.
But it may cause some more changes, e.g. _file_end in CompileLog should then also probably also be uint64_t.
We can look at that in another RFE.